cli/src/pcluster/cli/entrypoint.py

```diff
@@ -23,21 +23,27 @@
 import argparse
 from botocore.exceptions import NoCredentialsError  # TODO: remove
 
+# Suppress JSII warnings for Node version
+os.environ["JSII_SILENCE_WARNING_KNOWN_BROKEN_NODE_VERSION"] = "1"
+os.environ["JSII_SILENCE_WARNING_UNTESTED_NODE_VERSION"] = "1"
+os.environ["JSII_SILENCE_WARNING_DEPRECATED_NODE_VERSION"] = "1"
+os.environ["JSII_SILENCE_WARNING_END_OF_LIFE_NODE_VERSION"] = "1"
+
 # Controllers
-import pcluster.api.controllers.cluster_compute_fleet_controller
-import pcluster.api.controllers.cluster_instances_controller
-import pcluster.api.controllers.cluster_operations_controller
-import pcluster.api.controllers.image_operations_controller
-import pcluster.api.errors
-import pcluster.cli.commands.commands as cli_commands
-import pcluster.cli.logger as pcluster_logging
-import pcluster.cli.model
-from pcluster.api import encoder
-from pcluster.cli.commands.common import CliCommand, exit_msg, to_bool, to_int, to_number
-from pcluster.cli.exceptions import APIOperationException, ParameterException
-from pcluster.cli.logger import redirect_stdouterr_to_logger
-from pcluster.cli.middleware import add_additional_args, middleware_hooks
-from pcluster.utils import to_camel_case, to_snake_case
+import pcluster.api.controllers.cluster_compute_fleet_controller  # noqa: E402
```

Comment:
If I'm not wrong there is the possibility to suppress all of them with a single noqa line or by turning off/on the linter pre and after the imports.

Reply:
Apparently [it's not possible to apply it to a block of code](https://stackoverflow.com/questions/64428794/flake8-disable-linter-only-for-a-block-of-code).